meerkat_frontend/src/js/technical/tableManager.js

```diff
@@ -1209,150 +1209,292 @@ function drawAllClinicsCompleteness(containerID, regionID, locations, data) {
+
+		   }
+    ];
+    var data = [];
+
+    for (var i in cases) {
+      c = cases[i];
+
+      var datum = {
+        alert_id: c.variables.alert_id,
+        region: i18n.gettext(locations[c.region].name),
+        district: i18n.gettext(locations[c.district].name),
+        report_date: c.date.split("T")[0],
+
+        age: c.variables.agv_1
+
+      };
+      if ("ale_1" in c.variables) {
+        datum.investigation_date = c.variables.ale_1.split("T")[0];
+      }
+      if (c.clinic) {
+        datum.clinic = i18n.gettext(locations[c.clinic].name);
+      } else {
+        datum.clinic = i18n.gettext(c.type_name);
+      }
+      if (c.variables.pla_2) {
+        datum.profession = i18n.gettext(c.variables.pla_2);
+      } else {
+        datum.profession = "";
+      }
+      var gender = "";
+
+      if ("gen_1" in c.variables) {
+        datum.gender = i18n.gettext("Female");
+      } else if ("gen_2" in c.variables) {
+        datum.gender = i18n.gettext("Male");
+      }
+
+      status = i18n.gettext("Pending");
+      if ("ale_2" in c.variables) {
+        status = i18n.gettext("Confirmed");
+      } else if ("ale_3" in c.variables) {
+        status = i18n.gettext("Disregarded");
+      } else if ("ale_1" in c.variables) {
+        status = i18n.gettext("Ongoing");
+      }
+      datum.status = status;
+      status_2 = i18n.gettext("Alive");
+      if ("pla_3" in c.variables) {
+        status_2 = i18n.gettext("Dead");
+      }
+      datum.status_2 = status_2;
+      data.push(datum);
+    }
+    $('#' + containerID).html("<table> </table>");
+    $('#' + containerID + ' table').bootstrapTable({
+      columns: columns,
+      width: "100%",
+      data: data,
+      align: "center",
+      classes: "table table-hover",
+      pagination: true,
+      pageSize: 50
+    });
+    addPaginationListener('#' + containerID + ' table');
+  });
+}
 
-            status = i18n.gettext("Pending");
-            if ("ale_2" in c.variables) {
-                status = i18n.gettext("Confirmed");
-            } else if ("ale_3" in c.variables) {
-                status = i18n.gettext("Disregarded");
-            } else if ("ale_1" in c.variables) {
-                status = i18n.gettext("Ongoing");
-            }
-            datum.status = status;
-            status_2 = i18n.gettext("Alive");
-            if ("pla_3" in c.variables) {
-                status_2 = i18n.gettext("Dead");
-            }
-            datum.status_2 = status_2;
-            data.push(datum);
-        }
-        $('#' + containerID).html("<table> </table>");
-        $('#' + containerID + ' table').bootstrapTable({
-            columns: columns,
-            width: "100%",
-            data: data,
-            align: "center",
-            classes: "table table-hover",
-            pagination: true,
-            pageSize: 50
-        });
-        addPaginationListener('#' + containerID + ' table');
+function drawVHFTable(containerID, cases, variables) {
```

Comment:
Any chnce of a short comment explaing what a VHF table is? I presume we are not talking about VHF radio?

Reply:
Viral Haemorrhagic Fever